src/mon/PGMap.cc

```diff
@@ -1881,7 +1881,6 @@ void PGMap::dump_pool_stats(const OSDMap &osd_map, stringstream *ss,
     tbl.define_column("NAME", TextTable::LEFT, TextTable::LEFT);
     tbl.define_column("ID", TextTable::LEFT, TextTable::LEFT);
     if (verbose) {
-      tbl.define_column("CATEGORY", TextTable::LEFT, TextTable::LEFT);
       tbl.define_column("QUOTA OBJECTS", TextTable::LEFT, TextTable::LEFT);
```

Comment:
might need to update the test of `TEST_pool_quota` in `src/test/mon/test_pool_quota.sh`, line 39 and 40, so the `$3` and `$4` fields are extracted instead of `$4` and `$5` fields,

```
/home/jenkins-build/build/workspace/ceph-pull-requests/src/test/mon/test_pool_quota.sh:44: TEST_pool_quota:  '[' 0 '!=' N/A ']'
/home/jenkins-build/build/workspace/ceph-pull-requests/src/test/mon/test_pool_quota.sh:46: TEST_pool_quota:  return 1
/home/jenkins-build/build/workspace/ceph-pull-requests/src/test/mon/test_pool_quota.sh:24: run:  return 1
```

see https://jenkins.ceph.com/job/ceph-pull-requests/16213/consoleFull